draft-ietf-mls-protocol.md

```diff
@@ -1454,14 +1466,14 @@ to be used for its own sending chain:
            application_secret
                      |
                      V
-           Derive-Secret(., "app sender", [sender])
+           HKDF-Expand-Label(., "app sender", [sender], Hash.length)
                      |
                      V
            application_secret_[sender]_[0]
 ~~~
 
-Note that [sender] represent the uint32 value encoding the index
-of the participant in the ratchet tree.
+Note that [sender] represent the big-endian encoding of the uint32 word
+used to store the index of the participant in the ratchet tree.
```

Comment:
You could note here that this is the same as the TLS encoding of `sender` as a `uint32`.  Which is probably in fact how I would implement it.